File: Flavors/app/src/main/java/org/pcc/flavors/MainActivity.java
package org.pcc.flavors;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = (TextView) findViewById(R.id.textView);
        textView.setText("Hello No Flavor App");
    }
}

File: mvvm/app/src/main/java/org/pcc/arch_mvvm/view/MainActivity.kt
package org.pcc.arch_mvvm.view.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.Navigation.findNavController
import androidx.navigation.ui.NavigationUI.setupActionBarWithNavController
import org.pcc.arch_mvvm.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setupActionBarWithNavController(this,
            findNavController(this, R.id.main_navigation_frag)
        )
    }

    override fun onSupportNavigateUp() = findNavController(this, R.id.main_navigation_frag).navigateUp()
}
